backend/src/LeanCode.AppRating/CQRS/SubmitAppRatingCH.cs

```diff
@@ -38,20 +40,27 @@ public class SubmitAppRatingCH<TUserId> : ICommandHandler<SubmitAppRating>
 {
     private readonly IAppRatingStore<TUserId> store;
     private readonly IUserIdExtractor<TUserId> extractor;
+    private readonly IPublishEndpoint publishEndpoint;
 
-    public SubmitAppRatingCH(IAppRatingStore<TUserId> store, IUserIdExtractor<TUserId> extractor)
+    public SubmitAppRatingCH(
+        IAppRatingStore<TUserId> store,
+        IUserIdExtractor<TUserId> extractor,
+        IPublishEndpoint publishEndpoint
+    )
     {
         this.store = store;
         this.extractor = extractor;
+        this.publishEndpoint = publishEndpoint;
     }
 
     public Task ExecuteAsync(HttpContext context, SubmitAppRating command)
     {
+        TUserId userId = extractor.Extract(context);
         store
             .AppRatings
             .Add(
                 new AppRatingEntity<TUserId>(
```

Comment:
Nit: why the class has `Entity` in it's name?

Reply:
I just followed the convention and named the entity after `PushNotificationTokenEntity`

Reply:
I really regret introducing that convention (:sniffs: Hungarian notation :sniffs:). :(

Reply:
Yeah, that one we could/should get rid of, probably.